pilot/pkg/config/kube/ingress/conversion.go

```diff
@@ -90,6 +90,10 @@ func ConvertIngressV1alpha3(ingress v1beta1.Ingress, mesh *meshconfig.MeshConfig
 	}
 
 	for i, tls := range ingress.Spec.TLS {
+		if tls.SecretName == "" {
+			log.Infof("invalid ingress rule %s:%s for hosts %q, no secretName defined", ingress.Namespace, ingress.Name, tls.Hosts)
```

Comment:
Spammy logs, the better way is to validate it in an admission controller

Reply:
i agree that the better way is not to allow bad ingress with admission controller, but from Istio stability i would rather have noise in my logs than ingress controller dead. 
In this particular scenario  nginx or haproxy ingress controller would use default secret with the wildcard certificate, but if there is a secret, it will use it instead. 
So, I can't replace nginx/haproxy ingress with Istio K8s ingress in my existing clusters easily.

Reply:
in addition, what if secret is defined in k8s ingress resource, but it doesn't exist in K8s?